### PR TITLE
rename interfaces Go packages with a "proto" suffix

### PR DESCRIPTION
**Pull request by battlmonstr**
This depends on https://github.com/ledgerwatch/interfaces/pull/216

What's changed:
* added "proto" suffix to packages and generated folders (I've tried to keep folder names unchanged, but it was problematic for tooling)
* in the erigon code update package import paths, but keep them aliased to previous names such that it doesn't lead to massive code churn. We can do the code changes separately in subsequent PRs. Each code owner will do it for them.
* had to update make grpc command: tried various --go_opt=paths=xxx (see https://protobuf.dev/reference/go/go-generated/ ), but it still generated wrong namespaces, so had to list each of them separately

### CODE REVIEW

**Review by AskAlexSharov — APPROVED** [review]

**Comment by battlmonstr** [comment]
@AskAlexSharov do you have an open PR that makes erigon compatible with https://github.com/ledgerwatch/interfaces/pull/213 ? Could you tag me on it so that I follow it up when it is merged?

**Comment by AskAlexSharov** [comment]
> @AskAlexSharov Thanks for review. Do you have an open PR that makes erigon compatible with [ledgerwatch/interfaces#213](https://github.com/ledgerwatch/interfaces/pull/213) ? Could you tag me on it so that I follow it up when it is merged?

Giulio reverted this PR: https://github.com/ledgerwatch/interfaces/pull/214

File: erigon-lib/gointerfaces/downloader/mockgen.go [hunk 1]
```diff
@@ -1,3 +0,0 @@
-package downloader
-
-//go:generate mockgen -typed=true -destination=./downloader_client_mock.go -package=downloader . DownloaderClient
```
Comment (taratorio) [inline erigon-lib/gointerfaces/downloader/mockgen.go:3]: is this deleted by accident?

Comment (taratorio) [reply erigon-lib/gointerfaces/downloader/mockgen.go:3]: ah nvm I saw you have moved and renamed

**Review by taratorio — APPROVED** [review]

File: erigon-lib/gointerfaces/downloader/mockgen.go (hunk 1 above)
Comment (battlmonstr) [reply erigon-lib/gointerfaces/downloader/mockgen.go:3]: git was not so smart to detect this 😩

**Pull request merged**